backend-config/types.go

```diff
@@ -55,6 +55,8 @@ type DestinationT struct {
 	Transformations       []TransformationT
 	IsProcessorEnabled    bool
 	RevisionID            string
+	Account               AccountWithDefinition
+	DeleteAccount         AccountWithDefinition
```

Comment:
Absence of an `account` or `deleteAccount` should be using nil value semantics, not zero struct value, so that empty fields would be omitted from json altogether.
```suggestion
	Account               *AccountWithDefinition `json:"account,omitempty"`
	DeleteAccount         *AccountWithDefinition `json:"deleteAccount,omitempty"`
```